GenderPayGap.WebUI/Views/AccountCreation/ConfirmationPage.cshtml

```diff
@@ -26,23 +26,27 @@
     <div class="govuk-grid-column-two-thirds">
         
         <p class="govuk-body">
-            You are now able to add an organisation(s) to your user account and report their gender pay gap data.
+            You are now able to add your employer to your user account and
+            report their gender pay gap information. 
         </p>
         
-        <h2 class="govuk-heading-m">Editing your contact details or closing your account</h2>
+        <h2 class="govuk-heading-m">Editing your contact details or closing your user account</h2>
         
         <p class="govuk-body">
-            You can edit your contact details within Manage My Account.
+            You can edit your contact details within Manage Account.
         </p>
         <p class="govuk-body">
-            You will need to close your user account if:      
+            You will need to close your user account if you:
             <ul class="govuk-list govuk-list--bullet">
-                <li>You created a user account by mistake</li>
-                <li>You no longer work for, or manage the gender pay gap reporting for the organisation added to your account</li>
+                <li>created a user account by mistake</li>
+                <li>
+                    you no longer work for, or manage the gender pay gap reporting for
+                    the employer added to your account
+                </li>
             </ul>
         </p>
         <p class="govuk-body">
-            To complete the process for gender pay gap reporting please continue.
+            To complete the process for gender pay gap reporting please sign in to your account.
```

Comment:
Will the user not already be signed in to their account at this point?

Again, we could link this 'sign in to your account' text to the sign in page.

Reply:
No it doesn't log in when verifying the email, it just has a button link to the sign in page below it. Not sure if it is worth adding a second link?

Reply:
Ok cool - in that case I think the button is fine then (and we don't need to link here as well)